test/exception-handling.wast

```diff
@@ -59,23 +59,4 @@
       )
     )
   )
-
-  ;; Test subtype relationship
-  (func $subtype_test
-    (try
-      (do)
-      (catch
-        (drop (exnref.pop))
-        (drop
-          (block $l0 (result i32)
-            (rethrow
-              (br_on_exn $l0 $e0 (ref.null))
-            )
-          )
-        )
-      )
-    )
-
-    (throw $e1 (ref.null))
-  )
```

Comment:
It might be better to comment these tests out with a TODO comment so that we still have a record of them in the source.

Reply:
We are already extensively testing subtyping relationship in `reference-types.wast`, and this test didn't look very useful with no subtypes of `exnref` left (previously had one, `nullref`), so I deleted it. Shall I add it back and comment out nonetheless?

Reply:
Ah, that makes sense. I'm fine leaving this out, then 👍